src/Scientist/Internals/Experiment.cs

```diff
@@ -68,19 +69,19 @@ public ExperimentInstance(string name, Func<Task<T>> control, Func<Task<T>> cand
 
         static async Task<ExperimentResult> Run(Func<Task<T>> experimentCase)
         {
+            Chrono chrono = new Chrono();
             try
             {
-                // TODO: Refactor this into helper function?  
-                Stopwatch sw = new Stopwatch();
-                sw.Start();
-                var result = await experimentCase();
-                sw.Stop();
-
-                return new ExperimentResult(result, new TimeSpan(sw.ElapsedTicks));
+                T result;
+                using (Chronometer.New(out chrono))
+                {
+                    result = await experimentCase();
+                }
+                return new ExperimentResult(result, chrono.Timespan);
             }
             catch (Exception e)
             {
-                return new ExperimentResult(e, TimeSpan.Zero);
+                return new ExperimentResult(e, chrono.Timespan);
```

Comment:
I have changed the behavior here. 
It will allow us to time how long it takes to throw an (expected?) exception